apps/mudita-center-e2e/src/page-objects/overview-kompakt.page.ts

```diff
@@ -4,8 +4,52 @@
  */
 
 import { ChainablePromiseElement } from "webdriverio"
-import OverviewPage from "./overview.page"
+import { OverviewPage } from "./overview.page"
 
-class OverviewKompaktPage extends OverviewPage {}
+class OverviewKompaktPage extends OverviewPage {
+  public get aboutYourDevice() {
+    return $("p*=About your device")
+  }
 
+  public get kompaktImageElement() {
+    return $(
+      "#app > div.sc-hrCmsx.iRrrLX > div.sc-gtWJRm.fLIlvh > div.box-sizing-wrapper > div > div > div.sc-eGXPLf.bfaTWN > div > div.sc-eGXPLf.eAMdLd > img"
```

Comment:
These classes are changing every build. You can't do that. Ask for data-testid or try xpath

Reply:
updated to //div[@data-testid="generic-view-image"]